login_all_company/templates/assets.xml

```diff
@@ -1,13 +1,11 @@
 <?xml version="1.0" encoding="utf-8" ?>
 <odoo>
-  <data>
     <template id="assets_backend" name="account assets" inherit_id="web.assets_backend">
       <xpath expr="." position="inside">
         <script
```

Comment:
Indent this to 4 spaces as you are touching the file.

Reply:
Done!